SEACAS: SuplibCpp/Ioss: error w/ nvcc on vortex

## Bug Report
@trilinos/seacas

### Description
I get the following error while building w/ the latest Trilinos develop + SEACAS=ON on vortex:
```
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h: In function ‘fmt::v6::internal::fp fmt::v6::internal::normalize(fmt::v6::internal::fp)’:
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h:378:43: error: ‘significand_type’ was not declared in this scope
       static const int significand_size = bits<uint64_t>::value;
                                           ^~~~~~~~~~~~~~~~
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h:378:43: note: suggested alternative: ‘significand_size’
       static const int significand_size = bits<uint64_t>::value;
                                           ^~~~~~~~~~~~~~~~
                                           significand_size
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h:378:59: error: template argument 1 is invalid
       static const int significand_size = bits<uint64_t>::value;
                                                           ^
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/core.h: In member function ‘void fmt::v6::internal::arg_map<Context>::push_back(fmt::v6::internal::value<Context>) [with Context = fmt::v6::basic_format_context<std::back_insert_iterator<fmt::v6::internal::buffer<char> >, char>]’:
.../Trilinos/packages/seacas/libraries/suplib_cpp/fmt/core.h:1079:6: note: the ABI of passing aggregates with 16-byte alignment has changed in GCC 5
     void push_back(value<Context> val)
      ^~~~~~~~~
```
I didn't see an issue in the code so I'm not sure why the compiler doesn't like it. It looks like there was an attempt to fix this w/ https://github.com/trilinos/Trilinos/commit/417e2ebef09041f8ccd6effb9c9f4141ce0302c9 . I got past it by replacing `significand_type` but that's probably not the cleanest fix:
```
diff --git a/packages/seacas/libraries/ioss/src/fmt/format-inl.h b/packages/seacas/libraries/ioss/src/fmt/format-inl.h
index bd6c6bfdde..fb0a94f0c3 100644
--- a/packages/seacas/libraries/ioss/src/fmt/format-inl.h
+++ b/packages/seacas/libraries/ioss/src/fmt/format-inl.h
@@ -360,7 +360,7 @@ namespace internal {
     significand_type f;
     int              e;
 
-    static FMT_CONSTEXPR_DECL const int significand_size = bits<significand_type>::value;
+    static FMT_CONSTEXPR_DECL const int significand_size = bits<uint64_t>::value;
 
     fp() : f(0), e(0) {}
     fp(uint64_t f_val, int e_val) : f(f_val), e(e_val) {}
diff --git a/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h b/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h
index bd6c6bfdde..fb0a94f0c3 100644
--- a/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h
+++ b/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h
@@ -360,7 +360,7 @@ namespace internal {
     significand_type f;
     int              e;
 
-    static FMT_CONSTEXPR_DECL const int significand_size = bits<significand_type>::value;
+    static FMT_CONSTEXPR_DECL const int significand_size = bits<uint64_t>::value;
 
     fp() : f(0), e(0) {}
     fp(uint64_t f_val, int e_val) : f(f_val), e(e_val) {}
```

### Steps to Reproduce
1. SHA1:  1c59af4fdbfe99ed74fa569d3c96958868e4e84a
1. Configure script:  Can provide if needed.
1. Configure log:  Can provide if needed.
1. Build log:  Can provide if needed.
1. compilers/tpls:
gcc/7.2.1
spectrum-mpi/2019.06.24
cuda/10.1.243
boost-1.72.0
hdf5-1.10.6
lapack-3.9.0
netcdf-c-4.7.3
pnetcdf-1.12.1
zlib-1.2.11

--- a/packages/seacas/libraries/ioss/src/fmt/format-inl.h
+++ b/packages/seacas/libraries/ioss/src/fmt/format-inl.h
@@ -347,7 +347,6 @@ namespace internal {
   class fp
   {
   private:
-    using significand_type = uint64_t;
 
     // All sizes are in bits.
     // Subtract 1 to account for an implicit most significant bit in the
@@ -357,10 +356,10 @@ namespace internal {
     static FMT_CONSTEXPR_DECL const uint64_t implicit_bit = 1ull << double_significand_size;
 
   public:
-    significand_type f;
+    uint64_t f;
     int              e;
 
-    static FMT_CONSTEXPR_DECL const int significand_size = bits<significand_type>::value;
+    static FMT_CONSTEXPR_DECL const int significand_size = bits<uint64_t>::value;
 
     fp() : f(0), e(0) {}
     fp(uint64_t f_val, int e_val) : f(f_val), e(e_val) {}
@@ -445,7 +444,7 @@ namespace internal {
       assign(d);
       constexpr int min_normal_e =
           std::numeric_limits<float>::min_exponent - std::numeric_limits<double>::digits;
-      significand_type half_ulp =
+      uint64_t half_ulp =
           1 << (std::numeric_limits<double>::digits - std::numeric_limits<float>::digits - 1);
       if (min_normal_e > e)
         half_ulp <<= min_normal_e - e;

--- a/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h
+++ b/packages/seacas/libraries/suplib_cpp/fmt/format-inl.h
@@ -347,8 +347,6 @@ namespace internal {
   class fp
   {
   private:
-    using significand_type = uint64_t;
-
     // All sizes are in bits.
     // Subtract 1 to account for an implicit most significant bit in the
     // normalized form.
@@ -357,10 +355,10 @@ namespace internal {
     static FMT_CONSTEXPR_DECL const uint64_t implicit_bit = 1ull << double_significand_size;
 
   public:
-    significand_type f;
+    uint64_t f;
     int              e;
 
-    static FMT_CONSTEXPR_DECL const int significand_size = bits<significand_type>::value;
+    static FMT_CONSTEXPR_DECL const int significand_size = bits<uint64_t>::value;
 
     fp() : f(0), e(0) {}
     fp(uint64_t f_val, int e_val) : f(f_val), e(e_val) {}
@@ -445,7 +443,7 @@ namespace internal {
       assign(d);
       constexpr int min_normal_e =
           std::numeric_limits<float>::min_exponent - std::numeric_limits<double>::digits;
-      significand_type half_ulp =
+      uint64_t half_ulp =
           1 << (std::numeric_limits<double>::digits - std::numeric_limits<float>::digits - 1);
       if (min_normal_e > e)
         half_ulp <<= min_normal_e - e;

